src/main/java/org/folio/rest/impl/DelayedTasksAPI.java

```diff
@@ -31,4 +38,55 @@ public void deleteDelayedTaskExpiredMessages(String expirationDate, String statu
       .otherwise(this::mapExceptionToResponse)
       .onComplete(resultHandler);
   }
+
+  @Override
+  public void postDelayedTaskRetry(Map<String, String> okapiHeaders,
+    Handler<AsyncResult<Response>> asyncResultHandler, Context vertxContext) {
+
```

Comment:
Return response immediately

Reply:
Based on current implementation our repo it would be incredibly difficult to write test for that functional. I left response after.